litmuschaos/README.md

```diff
@@ -0,0 +1,28 @@
+# Integration of Kyverno with Litmus
+
+Kyverno is a policy engine designed for Kubernetes. It can validate, mutate, and generate configurations using admission controls and background scans. Litmus provides a large number of experiments for testing containers, pods, nodes, as well as specific platforms and tools. The advantage of chaos engineering is that one can quickly figure out issues that other testing layers cannot easily capture. This can save a lot of time in the future, and will help to find the loopholes in the system and fix them.
+
+## Steps to Execute LitmusChaos Experiment
+
+For running a LitmusChaos experiment following steps need to be followed - 
+
+#### Pulling Docker Image 

```

Comment:
Can we add "prerequisite" here to guide the user to install Kyverno using this image `ghcr.io/kyverno/kyverno:test-litmuschaos`? We can also attach the instructions to build new images against the latest code.

And let's make a note that "We'll change the base image soon so that the Litmuschaos tests can be run against the official images."